<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>W_VetoMessageBox</class>
 <widget class="QWidget" name="W_VetoMessageBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Niagara Solid</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color:white;
}</string>
     </property>
     <property name="text">
      <string>Le Directeur ne souhaite pas voter la prochaine loi
Accepter la demande de veto ?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>30</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonYes">
       <property name="font">
        <font>
         <family>Niagara Solid</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color:rgba(250,250,250,50);
	color:white;
}
QPushButton::hover{
	background-color:rgba(250,250,250,80);
}
QPushButton::pressed{
	background-color:rgba(250,250,250,100);
}</string>
       </property>
       <property name="text">
        <string>Oui</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNo">
       <property name="font">
        <font>
         <family>Niagara Solid</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color:rgba(250,250,250,50);
	color:white;
}
QPushButton::hover{
	background-color:rgba(250,250,250,80);
}
QPushButton::pressed{
	background-color:rgba(250,250,250,100);
}</string>
       </property>
       <property name="text">
        <string>Non</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
